src/components/ComponentList.js

```diff
@@ -146,6 +147,12 @@ export default class ComponentList extends React.Component {
               </Button>,
               'Revert Changes of this Definition'
             )}
+          <CopyUrlButton
+            route={ROUTE_DEFINITIONS}
+            path={component.toPath()}
+            bsStyle="default"
```

Comment:
We should also check if the button is actually following the other button's style.
If you had the chance to check, those are basically an Icon Button with a Tooltip.
Is this button the same?

Reply:
<img width="1027" alt="screen shot 2018-10-05 at 10 31 30" src="https://user-images.githubusercontent.com/17598688/46524675-dede8580-c889-11e8-8a49-c7cbd53e95ad.png">

I moved the button before the Revert Changes button, about the style is it ok like this?